docs/pages/_document.js

```diff
@@ -26,6 +26,7 @@ class GestaltDocument extends Document {
     return (
       <Html lang="en" dir={dir}>
         <Head>
+          <meta name="p:domain_verify" content="752e3976762ef39258186e60a40bbe5a" />
```

Comment:
@ponori Can you give more context for this? Why are we adding this?

Reply:
Yeah, absolutely. I'm adding this tag to connect our Gestalt Pinterest account to the site so we can officially claim it.
<img width="874" alt="Screen Shot 2022-06-01 at 11 28 32 AM" src="https://user-images.githubusercontent.com/50343812/171476588-5a994f80-9e90-4b22-9050-c46e15b2051c.png">